rpc_jobs/periodic_cleanup.yml

```diff
@@ -72,7 +73,7 @@
         // common.use_node/shared_slave/internal_slave.
         container.inside {
           stage("Docker Checkout"){
-            git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
+            git branch: env.RPC_GATING_BRANCH, url: common.RPC_GATING_REPO
```

Comment:
Nope, fixed. Was experimenting with constants in the common module, but forgot to remove that one.